src/components/tasks/card/SuggestionBox.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useEffect, useState } from 'react';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+import classNames from '@/components/tasks/card/suggestion.module.scss';
+import fetch from '@/helperFunctions/fetch';
+import { userDataType } from '@/interfaces/user.type';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+
+type Props = {
+    assigneeName: string;
+    showSuggestion: boolean;
+    onClickName: (userName: string) => void;
+};
+
+const SuggestionBox: FC<Props> = ({
+    assigneeName,
+    showSuggestion = false,
+    onClickName = () => undefined,
```

Comment:
why returns undefined ?

Reply:
It does not return anything but is just changing the state in the parent prop.

Reply:
and for the 'type Props' object, yes this is the convention we are following throughout the project.
@vvaibhavdesai

Reply:
for the most part, consisting of typeScript, I am just looking around the repo and figuring out. This is the first time i am working with it @vvaibhavdesai

Reply:
use ()=>void instead of undefined

Reply:
![image](https://github.com/Real-Dev-Squad/website-status/assets/64726586/06c98804-4962-403a-a027-983cd3bae5d0)
you dont need to do this 
just write `onClickName`